x-pack/functionbeat/docs/howto/howto.asciidoc

```diff
@@ -0,0 +1,38 @@
+[[howto-guides]]
+[role="xpack"]
```

Comment:
Is this right?

Reply:
Yah Functionbeat requires a basic license.